python/src/scipp/compat/mantid.py

```diff
@@ -452,7 +452,11 @@ def from_mantid(workspace, **kwargs):
     # workspace? This is not handled by ExtractMonitors above, I think.
     if monitor_ws is None:
         if hasattr(workspace, 'getMonitorWorkspace'):
```

Comment:
Is the `hasattr` still useful/required?

Reply:
Could be removed, but then would fail as an `AttributeError` for different workspace types. Would mean then using base exception type. So don't think that's worth doing.